python/tvm/meta_schedule/cost_model/xgb_model.py

```diff
@@ -217,6 +219,8 @@ class XGBConfig(NamedTuple):
     nthread : Optional[int],
         The number of threads to use.
         Default is None, which means to use physical number of cores.
+    tree_method str :
```

Comment:
Typo, might be changed to `tree_method: str`.

Reply:
Also, can you please specify the options in the doc? For example what methods are supported here? In that case you can use `Literal["auto", "...", "..."]` (replace the dots with other options) as the type annotation.

Reply:
Thanks a lot, I will fix it.